eng/Version.Details.xml

```diff
@@ -16,37 +16,37 @@
       <Uri>https://github.com/dotnet/emsdk</Uri>
       <Sha>d3583522209829d1ed0440662ba136c7b7700b16</Sha>
     </Dependency>
-    <Dependency Name="runtime.linux-arm64.Microsoft.NETCore.Runtime.JIT.Tools" Version="19.0.0-alpha.1.24366.1">
+    <Dependency Name="runtime.linux-arm64.Microsoft.NETCore.Runtime.JIT.Tools" Version="19.0.0-alpha.1.24367.1">
```

Comment:
these don't contain the windows changes, you need to watch https://dev.azure.com/dnceng/internal/_build/results?buildId=2497225&view=results :)